examples/kuka_iiwa_arm/iiwa_controller_demo.cc

```diff
@@ -25,68 +17,126 @@
 #include "systems/controllers/endeffector_velocity_controller.h"
 #include "systems/controllers/endeffector_position_controller.h"
 
+#include <nlohmann/json.hpp>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+
+using json = nlohmann::json;
 namespace dairlib {
 
+// CsvVector class: Takes CSV file as a parameter,
+// loads data into a 2D vector of doubles, provides
+// method to get the 2D vector.
+class CsvVector {
```

Comment:
Is there a reason for this to be a class, rather than just a method that returns the data vector?